File: Services/CrosswordStateService.cs
using System.Collections.Generic;
using Crossword.Objects;

namespace Crossword.Services;

public class CrosswordStateService : ICrosswordStateService
{
    public List<Word> WordsGrid { get; } = new();
    public List<Dictionary> Dictionaries { get; } = new();
    public List<Cell> EmptyCells { get; } = new();
}

File: crossword_python/services/CrosswordStateService.py
from crossword_python.models.Word import Word
from crossword_python.models.Dictionary import Dictionary
from crossword_python.models.Cell import Cell

class CrosswordStateService:
    """Хранилище состояния кроссворда"""
    def __init__(self):
        self.words_grid: list[Word] = [ ]
        self.dictionaries: list[Dictionary] = [ ]
        self.empty_cells: list[Cell] = [ ]